lib/elixir/lib/macro.ex

```diff
@@ -982,7 +984,7 @@ defmodule Macro do
       #=> "one + two"
 
   """
-  @deprecated "Use Macro.to_string/1 instead"
+  @deprecated "Use `Macro.to_string/1` instead"
```

Comment:
It seems we don't use backticks in deprecation messages. It makes sense, as those also shown up in editors, CLI and what not.

```suggestion
  @deprecated "Use Macro.to_string/1 instead"
```